apps/mobile/src/app/components/SignIn/SignInTemplate.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+import { PixelRatio, StyleSheet } from 'react-native';
+
+import { Box, Image } from '@inheartive/ui/atoms';
+import { LoginFormControl } from '@inheartive/ui/molecules';
+import { View } from 'native-base';
+import logo from '../../assets/logo.jpg';
+
+/* eslint-disable-next-line */
+export interface LoginTemplateProps {}
```

Comment:
Can "disable" be removed now? We could remove empty props too as we discussed recently.